util/test/sub_test

```diff
@@ -257,7 +257,7 @@ def ReadFileWithComments(f, ignoreLeadingSpace=True):
     if os.access(f, os.X_OK):
         try:
             # grab the chplenv so it can be stuffed into the subprocess env
-            env_cmd = [os.path.join(utildir, 'printchplenv'), '--simple']
+            env_cmd = [os.path.join(utildir, 'printchplenv'), '--all', '--simple', '--no-tidy']
```

Comment:
I think this should include --internal, in case some test wants a prediff to check CHPL_JEMALLOC (say) or some other internal variable.